pull_request_template.md

```diff
@@ -0,0 +1,16 @@
+### Description
+
+_Give your PR a **concise** yet **descriptive** title_
+_Please explain the changes you made here._
+_Explain the **motivation** for making this change. What existing problem does the pull request solve?_
+_Mention any issues fixed, addressed or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
```

Comment:
```suggestion
_Mention any issues fixed, addressed, or otherwise related to this pull request, including issue numbers or hard links for issues in other repos._
```